pkg/validator/schema.go

```diff
@@ -251,6 +253,10 @@ func applySchemaCheck(conf *config.Configuration, checkID string, test schemaTes
 	} else if check.Target == config.TargetPod {
 		passes, issues, err = check.CheckPod(test.Resource.PodSpec)
 	} else if check.Target == config.TargetContainer {
+		containerIndex := funk.IndexOf(test.Resource.PodSpec.Containers, func(value corev1.Container) bool {
+			return value.Name == test.Container.Name
+		})
+		prefix = "/spec/template/spec/containers/" + strconv.Itoa(containerIndex)
```

Comment:
I think we need to make this dynamic based on the Kind. E.g. this works for deployments, but not CronJobs

Reply:
there is an interesting error for that path -
```
add operation does not apply: doc is missing path: "/spec/template/spec/containers/0/imagePullPolicy"
```
unless you use `spec/containers/` need to test for cronjobs to see how it works